ext/include/opentelemetry/ext/http/common/url_parser.h

```diff
@@ -34,7 +35,7 @@ class UrlParser
   std::string query_;
   bool success_;
 
-  UrlParser(std::string url) : url_(url), success_(true)
+  UrlParser(std::string url) : url_(std::move(url)), success_(true)
```

Comment:
> Parameter 'url' is passed by value and only copied once; consider moving it to avoid unnecessary copies

https://clang.llvm.org/extra/clang-tidy/checks/performance/unnecessary-value-param.html